packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -32,6 +33,7 @@ export class Runtime {
   public static readonly DotNetCore2 =  new Runtime('dotnetcore2.0',  RuntimeFamily.DotNetCore);
   public static readonly DotNetCore21 = new Runtime('dotnetcore2.1',  RuntimeFamily.DotNetCore);
   public static readonly Go1x =         new Runtime('go1.x',          RuntimeFamily.Go);
+  public static readonly Ruby25 =       new Runtime('ruby2.5',        RuntimeFamily.Ruby, { supportsInlineCode: true });
```

Comment:
@rix0rrr I've added `supportsInlineCode: true`.

I couldn't reproduce the error below again:

> ZipFile can only be used when Runtime is set to either of nodejs, nodejs4.3, nodejs6.10, nodejs8.10, python2.7, python3.6.

But it would be great if you can test it.

Reply:
Can you add a unit test for this? I'd prefer to have a test on it in any case.

Reply:
@rix0rrr sure! 4add360c043f7c55585da2a5a63b4b1d731b3bba